deploy/config.ts

```diff
@@ -53,10 +53,10 @@ export const getAppStackConfig = (appStage: AppStage): ApplicationStackProps =>
         cloudFrontBucketName: cloudFrontBucketNameConfig[appStage],
         aliasDomainName: ['orcaui.umccr.org', 'orcaui.prod.umccr.org'],
         reactBuildEnvVariables: {
-          VITE_METADATA_URL: `https://metadata.umccr.org`,
-          VITE_WORKFLOW_URL: `https://workflow.umccr.org`,
-          VITE_SEQUENCE_RUN_URL: `https://sequence.umccr.org`,
-          VITE_FILE_URL: `https://file.umccr.org`,
+          VITE_METADATA_URL: `https://metadata.prod.umccr.org`,
```

Comment:
Is there any reason for calling the `.prod` domain version?

Reply:
it seems only working for .prod
![image](https://github.com/user-attachments/assets/b84de4e2-7332-4f6a-8a9f-c5a38668dbf6)
![image](https://github.com/user-attachments/assets/80ed2a57-bf64-410a-8a31-e3a867d38476)

Reply:
![image](https://github.com/user-attachments/assets/d274e505-4e1e-4c1f-a623-6c491beb5fc8)
only have the prod record